source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,94 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Failure counts of errors occurring outside examples
+
+In previous versions of RSpec, when errors occurred outside of examples, they
+could potentially have side effects like preventing examples from running, or
+stopping nested example groups from being defined. To help debug those issues,
+RSpec now prints when those errors occur in the end of run summary, for
+example:
+
+~~~
+Finished in 0.00033 seconds (files took 0.14097 seconds to load)
+0 examples, 0 failures, 1 error occurred outside of examples
+~~~
+
+Special thanks to Jon Rowe for adding this feature.
+
+### Core: `config.fail_if_no_examples`
+
+As it currently stands RSpec will exit with code `0` indicating success if no
+examples are defined. This configuration option makes it possible to cause RSpec
+to exit with code `1` indicating failure, which is useful for CI environments.
+
+~~~ ruby
+RSpec.configure do |config|
+  config.fail_if_no_examples = true
+end
+~~~
+
+A special thanks to Ewa Czechowska for getting this in to core.
+
```

Comment:
Before moving on to the expectations changes, there's one more notable core changes, IMO: color is automatically enabled if RSpec is outputting to a TTY, so there's no longer a need to set `--color` to opt-in to this behavior.  If you really want color (even if the output isn't a TTY), you can pass `--force-color`, or `--no-color` to disable color output.  Can you add a section on that?